<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotometricCalibrationWidget</class>
 <widget class="QWidget" name="PhotometricCalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>235</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabCentral">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabVignette">
      <attribute name="title">
       <string>Vignette</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="labelCoef1"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Coefficient 1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Coefficient 2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelCoef2"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>r⁴</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Coefficient 3</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelCoef3"/>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="buttonRenderVignette">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Render vignette</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>r⁶</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>r²</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="4">
        <widget class="VignetteView" name="vignetteRenderWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>190</width>
           <height>190</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCameraResponse">
      <attribute name="title">
       <string>Camera Response</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Emor A</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelEmorA"/>
       </item>
       <item row="0" column="2" rowspan="5">
        <widget class="EmorView" name="cameraResponseRenderWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>190</width>
           <height>190</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Emor B</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelEmorB"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Emor C</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelEmorC"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Emor D</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="labelEmorD"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Emor E</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="labelEmorE"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VignetteView</class>
   <extends>QWidget</extends>
   <header>libvideostitch-gui/widgets/vignetteview.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EmorView</class>
   <extends>QWidget</extends>
   <header>libvideostitch-gui/widgets/emorview.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
